creator-node/src/index.js

```diff
@@ -17,16 +17,31 @@ const exitWithError = (...msg) => {
   process.exit(1)
 }
 
+const verifyDBConnection = async () => {
+  try {
+    logger.info('Verifying DB connection...')
+    await sequelize.authenticate() // runs SELECT 1+1 AS result to check db connection
+    logger.info('DB connected successfully!')
+  } catch (connectionError) {
+    exitWithError('Error connecting to DB: ', connectionError)
+  }
+}
+
 const runDBMigrations = async () => {
   try {
     logger.info('Executing database migrations...')
     await runMigrations()
     logger.info('Migrations completed successfully')
-  } catch (err) {
-    exitWithError('Error in migrations: ', err)
+  } catch (migrationError) {
+    exitWithError('Error in migrations: ', migrationError)
```

Comment:
supernit but i think logger.error(arg1, arg2) will automatically add a space btwn the string concat so adding a trailing space here and line 26 may be unnecessary but idk

Reply:
you are correct! I will remove ⭐

Reply:
but I will also ask you to rereview :P

Reply:
![image](https://user-images.githubusercontent.com/3323835/135333406-769f872f-8ad0-42a3-a86c-906362d2cc86.png)

Reply:
also feel free to merge, GH should let you 😎

Reply:
awesome - will do as soon as checks pass ✅ thanks for pointing this out! TIL